css/docs.css

```diff
@@ -73,7 +73,7 @@
 
 .doc-main-body{
     width: 80%;
-    height: 100%;
+    height: auto;
     background-color: transparent;
     overflow-y: auto;
```

Comment:
can use shorthand here

Reply:
By mistake I put 2 height their....I will correct it